src/xdocs/property_types.xml

```diff
@@ -368,22 +407,16 @@
        ...
     }
 
-    if (a > 0) {
+    if (a &#62; 0) {
        ...
-    <b>}</b> int i = 5; // this is NOT OK, next part of a multi-block statement is absent
-
-    // Single line blocks will rise violations, because right curly
-    // brace is not on the same line as the next part of a multi-block
-    // statement, it just ends the line.
-    public long getId() {return id;<b>}</b> // this is NOT OK
+    <b>}</b> int i = 5; // NOT OK, no next part of a multi-block statement, so should be alone
 
     Thread t = new Thread(new Runnable() {
        @Override
        public void run() {
                   ...
-       <b>}</b> // this is NOT OK, brace is not on the same line as the next part
-         // of a multi-block statement
-    <b>}</b>); // this is OK, allowed for better code readability
+       <b>}</b> // this is OK, should be alone as next part of a multi-block statement is absent
+    <b>}</b>); // this is OK, this case is out of scope of RightCurly Check (see issue #5945)
```

Comment:
> > This should also include line 419 to say that we consider end semicolons, parenthesis, etc... as part of the multi-block statement.

> this case is out of scope of RightCurly Check 

Should we be saying `OK` or should we say something like `this is ignored` or some such?
We say possible violation case is `OK`, seems to me we should use different wording for area that check won't even look at.

Reply:
not sure what will be best ... just removed "this is ok".